src/main/webapp/resources/js/pages/analysis/components/tree/Tree.jsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * This file renders the tree component
+ */
+
+import React, { useContext, useEffect, useState } from "react";
+import { getI18N } from "../../../../utilities/i18n-utilities";
+import { TabPaneContent } from "../../../../components/tabs/TabPaneContent";
+import Phylocanvas from "react-phylocanvas";
+import { Button, Radio } from "antd";
+import { AnalysisContext } from "../../../../contexts/AnalysisContext";
+import { getNewickTree } from "../../../../apis/analysis/analysis";
+import { ContentLoading } from "../../../../components/loader/ContentLoading";
+import { WarningAlert } from "../../../../components/alerts/WarningAlert";
+import styled from "styled-components";
+import { SPACE_MD, SPACE_XS } from "../../../../styles/spacing";
+import { BORDER_LIGHT } from "../../../../styles/borders";
+
+const CANVAS_HEIGHT = 600;
+
+const VisualizationWrapper = styled.div`
+  height: ${CANVAS_HEIGHT}px;
+  border: ${BORDER_LIGHT};
+`;
+
+const ButtonGroupWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: ${SPACE_MD};
+`;
+
+export default function Tree() {
+  const [newickString, setNewickString] = useState(null);
+  const [serverMsg, setserverMsg] = useState(null);
```

Comment:
Can you make this proper camelCase: `setServerMsg`

Reply:
Fixed in cba6f9588a746bd177bea92e49a052caa6586fe5